test/app/views/core/SubscribeModal.spec.coffee

```diff
@@ -193,18 +192,19 @@ describe 'SubscribeModal', ->
           expect(@getTrackerEventNames()).toDeepEqual(
             [ "Start Lifetime Purchase", "Fail Lifetime Purchase" ])
 
-    describe "when user's country has regional pricing", ->
-      beforeEach ->
-        me.set({_id: '1234', country: 'brazil'})
-        @purchaseRequest = jasmine.Ajax.stubRequest('/db/products/3/purchase')
-        @modal = new SubscribeModal({products: new Products(productListInternational)})
-        @modal.render()
-        jasmine.demoModal(@modal)
-        @openAsync.and.returnValue(tokenSuccess)
-      afterEach ->
-        me.set({country: undefined})
+      describe "when user's country has regional pricing", ->
+        beforeEach ->
+          me.set({_id: '1234', country: 'china'})
+          @purchaseRequest = jasmine.Ajax.stubRequest('/db/products/3/purchase')
+          @modal = new SubscribeModal({products: new Products(productListInternational)})
+          @modal.render()
+          jasmine.demoModal(@modal)
+          @openAsync.and.returnValue(tokenSuccess)
+        afterEach ->
+          me.set({country: undefined})
+
+        it 'uses Stripe', ->
+          expect(@modal.$('.stripe-lifetime-button').length).toBe(1)
+          expect(@modal.$('#paypal-button-container').length).toBe(0)
+          expect(@payPalButton).toBeUndefined()
```

Comment:
I think these all got indented an extra two spaces, nesting this within the `"when user's country does not have regional pricing"` case?